tekton-listener/DEVELOPMENT.md

```diff
@@ -0,0 +1,20 @@
+# Tekton Listener development guide
+
+## Setup instructions
+
+```bash
+export KO_DOCKER_REPO='gcr.io/my-gcloud-project-name'
+ko apply -f config/
+```
+
+### Minikube
+
+To dev/test locally with minikube:
+
+* Get the `ko` command: `go get -u github.com/google/ko/cmd/ko`
+* Load your docker environment vars: `eval $(minikube docker-env)`
+* Start a registry: `docker run -it -d -p 5000:5000 registry:2`
```

Comment:
Using `KO_DOCKER_REPO=ko.local` makes it possible to not use a registry (it will do a `docker load` on the minikube docker daemon)